formulas/mo/files/mo

```diff
@@ -75,7 +75,7 @@ mo() (
                         ;;
 
                     --source=*)
-                        f2source="${1#--source=}"
+                        f2source="${arg#--source=}"
```

Comment:
That change was a bugfix to mo. https://github.com/tests-always-included/mo/commit/ecfc973c2f60906372024fe3e6a83c5ea8cfb95c